### PR TITLE
Updated hopin workshop invitation

### PR DESCRIPTION
**Pull request by raukadah**

### CODE REVIEW

File: templates/emails/hopin-workshops [hunk 1]
```diff
@@ -2,28 +2,29 @@
...
-- [Hands-On: Containerizing Python and Deploying to IoT and Edge Devices](https://in.pycon.org/cfp/2020/proposals/hands-on-containerizing-python-and-deploying-to-iot-and-edge-devices~egJYk/) David T & Vipul Gupta
-- [Python on Kubernetes: A primer](https://in.pycon.org/cfp/2020/proposals/python-on-kubernetes-a-primer~aOY7Y/) Nabarun Pal
-- [Animating Data in PowerPoint](https://in.pycon.org/cfp/2020/proposals/animating-data-in-powerpoint~dyR6e/) Anand S
-- [Concurrency Patterns with Python](https://in.pycon.org/cfp/2020/proposals/concurrency-patterns-with-python~e9r8B/) Anmol Jindal
+* [Building Powerful Interactive AI/ML web apps using Streamlit](https://in.pycon.org/cfp/2021/proposals/building-powerful-interactive-aiml-web-apps-using-streamlit~e316x/)  Sasidhar Donaparthi
+* [Getting Kafka-esque with Python] (https://in.pycon.org/cfp/2021/proposals/getting-kafka-esque-with-python~elYRr/) Dipen Chawla, Devesh Bajaj
+* [Make your Cloud Native Python applications Kubernetes native!](https://in.pycon.org/cfp/2021/proposals/make-your-cloud-native-python-applications-kubernetes-native~bmZqr/) Nabarun Pal
```
Comment (palnabarun) [inline templates/emails/hopin-workshops:17]:
```suggestion
* [Make your Python applications Kubernetes native!](https://in.pycon.org/cfp/2021/proposals/make-your-python-applications-kubernetes-native~bmZqr/) Nabarun Pal
```

**Review by palnabarun — CHANGES_REQUESTED** [review]
just one change

File: templates/emails/hopin-workshops [hunk 2]
```diff
@@ -2,28 +2,29 @@
...
+* [Introduction to Property-Based Testing](https://in.pycon.org/cfp/2021/proposals/introduction-to-property-based-testing~elY5r/) Zac Hatfield-Dodds
 
 Please make sure you have set up your system correctly with the prerequisites provided in the workshop description.
 
-We hope you have a fun learning experience at the workshops. If you have any problems or queries during the workshops, please reach out to the Help Desk on [Zulip](https://pyconindia.zulipchat.com/#narrow/stream/258750-2020.2Fhelpdesk) or Hopin.
+We hope you have a fun learning experience at the workshops. If you have any problems or queries during the workshops, please reach out to the Help Desk on [Zulip](https://pyconindia.zulipchat.com/#narrow/stream/300363-2021.2Fhelpdesk) or Hopin.
 
-Please make sure you follow the conference [Code of Conduct](https://in.pycon.org/2020/coc/) to make the conference a great experience for everyone.
+Please make sure you follow the conference [Code of Conduct](https://in.pycon.org/2021/coc/) to make the conference a great experience for everyone.
```
Comment (vipulgupta2048) [inline templates/emails/hopin-workshops:26]:
```suggestion
Please make sure you follow the conference [Code of Conduct](https://in.pycon.org/2021/coc/) to make PyCon India a great experience for everyone.
```

File: templates/emails/hopin-workshops [hunk 3]
```diff
@@ -2,28 +2,29 @@
...
-- [Creating real-world simulations using Numpy, SciPy, and Matplotlib](https://in.pycon.org/cfp/2020/proposals/creating-real-world-simulations-using-numpy-scipy-and-matplotlib~e0Rgy/) Ashok Govindarajan
-- [Hands-On: Containerizing Python and Deploying to IoT and Edge Devices](https://in.pycon.org/cfp/2020/proposals/hands-on-containerizing-python-and-deploying-to-iot-and-edge-devices~egJYk/) David T & Vipul Gupta
-- [Python on Kubernetes: A primer](https://in.pycon.org/cfp/2020/proposals/python-on-kubernetes-a-primer~aOY7Y/) Nabarun Pal
-- [Animating Data in PowerPoint](https://in.pycon.org/cfp/2020/proposals/animating-data-in-powerpoint~dyR6e/) Anand S
-- [Concurrency Patterns with Python](https://in.pycon.org/cfp/2020/proposals/concurrency-patterns-with-python~e9r8B/) Anmol Jindal
+* [Building Powerful Interactive AI/ML web apps using Streamlit](https://in.pycon.org/cfp/2021/proposals/building-powerful-interactive-aiml-web-apps-using-streamlit~e316x/)  Sasidhar Donaparthi
+* [Getting Kafka-esque with Python] (https://in.pycon.org/cfp/2021/proposals/getting-kafka-esque-with-python~elYRr/) Dipen Chawla, Devesh Bajaj
```
Comment (vipulgupta2048) [inline templates/emails/hopin-workshops:16]:
```suggestion
* [Getting Kafka-esque with Python](https://in.pycon.org/cfp/2021/proposals/getting-kafka-esque-with-python~elYRr/) Dipen Chawla, Devesh Bajaj
```

File: templates/emails/hopin-workshops [hunk 4]
```diff
@@ -2,28 +2,29 @@
...
+* [Make your Python applications Kubernetes native!](https://in.pycon.org/cfp/2021/proposals/make-your-python-applications-kubernetes-native~bmZqr/) Nabarun Pal
+* [IndicNLP: Natural Language Processing for Indian Languages using Python] (https://in.pycon.org/cfp/2021/proposals/indicnlp-natural-language-processing-for-indian-languages-using-python~egJZ3/) Gajendra Deshpande
+* [Knowledge graph data modelling with TerminusDB] (https://in.pycon.org/cfp/2021/proposals/knowledge-graph-data-modelling-with-terminusdb~dJ6yP/) Cheuk Ting Ho
```
Comment (vipulgupta2048) [inline templates/emails/hopin-workshops:19]:
```suggestion
* [Knowledge graph data modeling with TerminusDB](https://in.pycon.org/cfp/2021/proposals/knowledge-graph-data-modelling-with-terminusdb~dJ6yP/) Cheuk Ting Ho
```

File: templates/emails/hopin-workshops [hunk 5]
```diff
@@ -2,28 +2,29 @@
 
 Hey {{ first name }},
 
-Welcome to the PyCon India 2020 Workshops. Thank you for choosing to attend our workshops for getting hands-on knowledge about things you've been looking to learn.
+Welcome to the PyCon India 2021 Workshops!
+Thank you for choosing to attend our workshops for getting hands-on knowledge about things you've been looking to learn.
 
-The workshops will be taking place on Hopin, which you can join using < >, and you can interact with other attendees and speakers on our Zulip channel [this.](https://pyconindia.zulipchat.com/join/it6vi5wfhfbdwr4kcn5mwa6c/)
+The Workshop will be kicking off on Sep 17th, 11:45 IST (UTC+5:30) with the opening address. 
+Join us for the event on Hopin using <magic link>. You can find the schedule on the Hopin event page at all times. Don't forget to join our communication platform [Zulip](https://pyconindia.zulipchat.com/join/ilavx7muukcacac6ftzcfypb/) to connect and network with other workshop attendees and speakers. Get the ice broken by saying hi! in the **#2021/discussions** Zulip stream.
```
Comment (vipulgupta2048) [inline templates/emails/hopin-workshops:9]:
```suggestion
Join us for the event on Hopin using {{event.link}}. You can find the schedule on the Hopin event page at all times. Don't forget to join our communication platform [Zulip](https://pyconindia.zulipchat.com/join/ilavx7muukcacac6ftzcfypb/) to connect and network with other workshop attendees and speakers. Get the ice broken by saying hi! in the **#2021/discussions** Zulip stream.
```

**Review by vipulgupta2048 — APPROVED** [review]

**Pull request merged**